test/c3/appframework/appframework_test.go

```diff
@@ -413,8 +413,8 @@ var _ = Describe("c3appfw test", func() {
 			testenvInstance.Log.Info("Verify Apps are installed Locally on CM and Deployer by running Splunk CLI commands for app", "version", appVersion)
 			testenv.VerifyAppInstalled(deployment, testenvInstance, testenvInstance.GetName(), podNames, appListV1, false, "enabled", false, false)
 
-			// Verify apps are not copied in /etc/master-apps/ on CM and /etc/shcluster/ on Deployer (therefore not installed on peers and on SH)
-			testenvInstance.Log.Info("Verify Apps are NOT copied to /etc/master-apps/ on CM and /etc/shcluster/ on Deployer for app", "version", appVersion, "App List", appFileList)
+			// Verify apps are not copied in /etc/manager-apps/ on CM and /etc/shcluster/ on Deployer (therefore not installed on peers and on SH)
```

Comment:
Is it really "/etc/manager-apps/" now? `splcommon.ManagerApps` still points to `"etc/master-apps"` in paths.go.  May be its better to modify the comment to remove the bias language